box-ui/src/components/ts/CaptureImage.tsx

```diff
@@ -147,6 +151,24 @@ const CaptureImage: FunctionComponent<CaptureImageProps> = (props: CaptureImageP
                     );
                 })}
             </Menu>
+            <IconButton id='TutoButton' aria-label='help' onClick={() => setDisplayFocus(!displayFocus)}>
+                <HelpIcon />
+            </IconButton>
+            {displayFocus ? (
```

Comment:
You should use 
```suggestion
            {displayFocus && (
```
to avoid the `: null`at the end